litestar/cli/commands/core.py

```diff
@@ -71,24 +71,25 @@ def info_command(app: Litestar) -> None:
 )
 @option("--host", help="Server under this host", default="127.0.0.1", show_default=True)
 @option(
+    "-fd",
     "--fd",
```

Comment:
Is the intent to remove all of the `--[shorthand]` options in favor of `-[shorthand]`?  I think we should do this before we leave beta if so.

Reply:
Most of this is because we have the long form commands (`--[command]`) but the shorthand versions are non-existent.

Click displays the shorthand appropriately when these are set, but without them we don't have that.

![image](https://github.com/litestar-org/litestar/assets/45884264/37a07c6c-8bb6-401b-bb13-99467ec60011)

Reply:
> Is the intent to remove all of the --[shorthand] options in favor of -[shorthand]? 

So the intent here is to properly set the shorthand commands in addition to their current long form versions.

current:
![image](https://github.com/litestar-org/litestar/assets/45884264/2b7400b6-5f95-4d94-baee-fafa9f615e35)

proposed:
(pictured in last comment)

Reply:
Also, wrt `--uds` and `--fd` in particular, these seem like they shouldve been `-uds` and `-fd` from the start?
At least that is the convention (that i am used to).. it is pretty odd (to me at least) to have multiple long form arguments instead of one short/one long.

Reply:
Actually, I don't think so? This can get confusing IMO when combining multiple shorthands, e.g, `-uds` + `-R` would be `-udsR`, which looks like it's 4 flags, when it's actually just 2.

The proper way to have these is to reserve the `-` option for single letter options, and use `--` for everything else.

Also, the already short option names don't really *need* a shorthand. Wdyt?

Reply:
I'm ok with either way we choose to go on this.  

The user who chooses to use this particular syntax `-udsR` would likely know they are using 2 flags instead of 4, so I think it's probably fine to overcomplicate it.